package.json

```diff
@@ -25,12 +33,16 @@
     "test:browser": "cross-env BABEL_ENV=test jest --testEnvironment=jsdom --testPathIgnorePatterns=test/node-only --testPathIgnorePatterns=cypress",
     "test:node": "cross-env BABEL_ENV=test jest --testEnvironment=node --testPathIgnorePatterns=test/browser-only --testPathIgnorePatterns=cypress",
     "test:types": "tsd",
+    "test:exports": "ls-exports path .",
```

Comment:
Cool tool that tests that your package.json is valid - https://github.com/ljharb/list-exports